CCDB/include/CCDB/Backend.h

```diff
@@ -8,7 +8,7 @@
 #include <string>
 #include <vector>
 
-#include "FairMQDevice.h"
+#include <FairMQDevice.h>
```

Comment:
But your project got a [rule](https://htmlpreview.github.io/?https://github.com/AliceO2Group/CodingGuidelines/blob/master/coding_guidelines.html#Names_and_Order_of_Includes) for it anyways.